README.md

```diff
@@ -477,16 +497,31 @@ class GreetingWorkflow:
     async def current_greeting(self) -> str:
         return self._current_greeting
 
+```
+
+This assumes there's an activity in `my_activities.py` like:
```

Comment:
nit: Why do activities need the `my_` prefix and workflows don't?

Reply:
I don't specify a workflow file name in this section. If you're talking about the quick start section, neither needs a `my_` prefix. I chose to add it here for clarity since it's not an end-to-end example.